### ISSUE TITLE
Add Myrient as a source

### ISSUE THREAD

**Issue by Vysp3r** [Vysp3r/RetroPlus#7]
Here's a link to the website: https://myrient.erista.me/

**Comment by Vysp3r** [comment]
@Snakeeater97 Do you know what directories I should add as source?
![image](https://github.com/Vysp3r/RetroPlus/assets/62019735/f905cc59-1718-41ea-b2b5-f80e32fc97b7)
The TOSEC stuff doesn't look like something I need to add and Miscellaneous either, but do I need to add both Redump and No-Intro?

**Comment by Snakeeater97** [comment]
> @Snakeeater97 Do you know what directories I should add as source? ![image](https://private-user-images.githubusercontent.com/62019735/303860102-f905cc59-1718-41ea-b2b5-f80e32fc97b7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc1OTcxOTQsIm5iZiI6MTcwNzU5Njg5NCwicGF0aCI6Ii82MjAxOTczNS8zMDM4NjAxMDItZjkwNWNjNTktMTcxOC00MWVhLWIyYjUtZjgwZTMyZmM5N2I3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjEwVDIwMjgxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5NGVhZGE1ZTI2YmZhZGNhZDVmNjM5MWVkNWI4OWNiY2YxZGI2OTc1ZjMyOGMwN2U4NDdjOGI4MzUxYjU0NWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.g1iC8lUihVKxplMfPoqM4OzPF7D74HbzGJ_gJWoGxuc) The TOSEC stuff doesn't look like something I need to add and Miscellaneous either, but do I need to add both Redump and No-Intro?

Yes, looks like both redump and no-intro have all the relevant files (I.e no intro doesn't have dreamcast but redump does)

**Comment by shadopsz** [comment]
Redump is usually focused ROMs taken from optical media like CDs (GameCube, PS, Dreamcast, etc), while No-Intro is generally cartridge based stuff (NES, SNES, GBC, etc).

**Comment by Vysp3r** [comment]
God damn. That website has so many systems. I don't know how much time that will take me to add it as a source. I could of course simply add the one vimm's has for now and create another issue to add the left over systems later. What do you guys think?

**Comment by shadopsz** [comment]
> God damn. That website has so many systems. I don't know how much time that will take me to add it as a source. I could of course simply add the one vimm's has for now and create another issue to add the left over systems later. What do you guys think?

I think that's the best approach. The stuff listed on Vimm covers like 90% of what people are looking for IMO

**Comment by Vysp3r** [comment]
I'll be working on adding that source as fast as possible since Vimm's Lair seems to be changing some stuff around to break my parser. Maybe it's just the way I made my shitty parser tho. I'll make the Myrient one in a better way and redo Vimm's one after that.
